src/dotnet-install.ps1

```diff
@@ -863,17 +881,29 @@ function DownloadFile($Source, [string]$OutPath) {
 
     $Stream = $null
 
+    $remoteFileSize = Get-Remote-File-Size -zipUri $Source
+    
     try {
         $Response = GetHTTPResponse -Uri $Source
         $Stream = $Response.Content.ReadAsStreamAsync().Result
         $File = [System.IO.File]::Create($OutPath)
         $Stream.CopyTo($File)
         $File.Close()
+
+        $fileSize = [long](Get-Item $OutPath).Length
+        Say "Downloaded file $Source size is $fileSize bytes."
     }
     finally {
         if ($null -ne $Stream) {
             $Stream.Dispose()
         }
+
+        if ((![string]::IsNullOrEmpty($remoteFileSize)) -and $remoteFileSize -ne $fileSize) {
```

Comment:
Add `-and ![string]::IsNullOrEmpty($fileSize)` so we don't print the message if anything in the try block failed. Or maybe better, move the new code inside the try block? There's no point executing this code if the download has failed.